docs/hub/endpoints.md

```diff
@@ -12,10 +12,11 @@ We have open endpoints that you can use to retrieve information from the Hub as
 | ------------------------------------------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
-| /api/models  <br/>  GET                                                              | Get information from all models in the Hub. You can specify additional parameters to have more specific results.  <br/>- `filter`: Filter based on tags, such as `text-classification` or `spacy`.<br/>- `sort`: Property to use when sorting. <br/>- `direction`: Direction in which to sort. <br/>- `limit`: Limit the number of models fetched. <br/>- `full`: Whether to fetch most model data, such as all tags, the files, etc. <br/>- `config`: Whether to also fetch the repo config. | `list_models()`                        | ```params= {   "filter":"filter", "full":"full", "sort": "sort",   "direction": "direction",   "limit": "limit",   "config": "config" }```                   |
+| /api/models  <br/>  GET                                                              | Get information from all models in the Hub. You can specify additional parameters to have more specific results.  <br/>- `author`: Filter models by an author or organization, such as `huggingface` or `microsoft`<br/>- `filter`: Filter based on tags, such as `text-classification` or `spacy`.<br/>- `sort`: Property to use when sorting. <br/>- `direction`: Direction in which to sort. <br/>- `limit`: Limit the number of models fetched. <br/>- `full`: Whether to fetch most model data, such as all tags, the files, etc. <br/>- `config`: Whether to also fetch the repo config. | `list_models()`                        | ```params= {   "filter":"filter", "full":"full", "sort": "sort",   "direction": "direction",   "limit": "limit",   "config": "config" }```                   |
```

Comment:
Should author also be added to the payload? Same below

Reply:
you can also document the `search` param (which is a substring search on the repo ids (including user names))